collection/playbooks/validate.yml

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ansible-playbook
+- hosts: localhost
+  gather_facts: false
+  collections:
+    - ssbarnea.molecule
+  tasks:
+
+    - name: Check if molecule_header filter can be used (1)
+      debug:
+        msg: "{{ '' | ssbarnea.molecule.molecule_header }}"
+
+    - name: Check if molecule_header filter can be used (2)
+      debug:
+        msg: "{{ '' | molecule_header }}"
```

Comment:
@bcoca Apparently it chokes here with:
```
fatal: [localhost]: FAILED! => {"msg": "template error while templating string: no filter named 'molecule_header'. String: {{ '' | molecule_header }}"}
```

Not sure how to convince Ansible to find the filter using short name.

Reply:
Docs specifically say you can only use short name for modules. Anything else always requires FQCN